scripts/datacontainer/Dockerfile

```diff
@@ -1,9 +1,7 @@
-FROM mcr.microsoft.com/cbl-mariner/distroless/base:1.0
-
-RUN apk update && \
-    apk add git
+FROM mcr.microsoft.com/oss/mirror/docker.io/library/ubuntu:20.04
 
```

Comment:
Can you add `RUN apt update`?

Reply:
We don't need this.  Because the ubuntu already have git. The whole Dockerfile doesn't have any other dependencies.